### ISSUE TITLE
Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by dungx48** [marcoslucianops/DeepStream-Yolo#329]
I converted my custom model yolov5 from .pt to onnx and engine. When i run "$deepstream-app -c /home/DeepStream-Yolo/deepstream_app_config.txt", something went wrong
![image](https://user-images.githubusercontent.com/77237744/231083910-351af1ea-3e0b-4705-b3c0-8f2733252838.png)

**Comment by dungx48** [comment]
I use docker Deepstream-6.1, tensorrt8.2.5.1, cuda 11.6

**Comment by Lvjinhong** [comment]
I also encountered the same error, but before that, I got an error because the model_b1_gpu0_fp32.engine file was missing. After I changed it to my own generated engine file and ran it again, the result was only:
Using winsys: x11
Deserialize yoloLayer plugin: (Unnamed Layer* 256) [PluginV2IOExt]
Segmentation fault (core dumped)

I don't know why this is happening? My versions are CUDA 10.2, JetPack 4.6.3, DeepStream 6.0.1, and I urgently need relevant support. Thank you!
